packages/runtime/container-runtime/src/test/opLifecycle/remoteMessageProcessor.spec.ts

```diff
@@ -310,4 +310,23 @@ describe("RemoteMessageProcessor", () => {
 			"unexpected processing of groupedBatch",
 		);
 	});
+
+	it("Processing empty groupedBatch works as expected", () => {
+		const groupedBatch = {
+			type: MessageType.Operation,
+			sequenceNumber: 10,
+			clientSequenceNumber: 12,
```

Comment:
side note - I don't this it's realistic/possible for CSN to be greater than seq num.